nixos/modules/system/boot/stage-1.nix

```diff
@@ -432,7 +432,7 @@ let
          }
 
         # mindepth 1 so that we don't change the mode of /
-        (cd "$tmp" && find . -mindepth 1 -print0 | sort -z | bsdtar --uid 0 --gid 0 -cnf - -T - | bsdtar --null -cf - --format=newc @-) | \
+        (cd "$tmp" && find . -mindepth 1 | xargs touch -amt 197001010000 && find . -mindepth 1 -print0 | sort -z | bsdtar --uid 0 --gid 0 -cnf - -T - | bsdtar --null -cf - --format=newc @-) | \
```

Comment:
you should use SOURCE_EPOCH_DATE here IMHO, cc @raboof (wdyt about that?)

Reply:
TBH I don't feel strongly in this case: since this is a `writeScriptBin` derivation, there isn't really a 'source', so 'the date of the source' is not really meaningful. I can live with either `SOURCE_DATE_EPOCH` or a constant like this.

Reply:
@RaitoBezarius do you want me to change it? I don't have strong feelings about this either.

Reply:
Since this is already an improvement over the status quo, I'd say this is fine.

Reply:
Yep, let's resolve.